### DISCUSSION TITLE
Linking / Merging transactions.

### DISCUSSION THREAD

**Discussion by bjmd** [firefly-iii/firefly-iii#6202] [General]
Hi all. I am in the beginning stages of testing Firefly iii and there is some functionality which I can't get my head around. Is it possible to link / merge transfer transactions between accounts?

I have imports setup from both Nordigen and Spectre with a bunch of different accounts in each and when I import I would like to merge transfer transactions. A simple example of what I mean: Say I transfer $100 from my current account to my savings account, but I have auto import for both accounts. I then have a rule in place which converts the withdrawal to a transfer (or deposit into transfer) and the transfers then appear in both accounts twice. I then begun looking for a way to merge these, with no luck. Some googling noted that they needed to be identical in order for the engine to match them, so I updated the rule to re-write the descriptions, the result is that in /debug the transactions appear to be exactly the same except the time stamps, id's and the hash. But the transactions still don't get merged. 

Am I miss understanding how this process should function? 

Thanks in advance. It's mighty impressive how good this software is for a free, open source project.

**Comment by JC5** [comment]
You can't, at least not in the way you want. You'll have to manage them manually unfortunately.

**Reply by bjmd** [reply]
Great thanks, good to know I wasn't completely missing something. 

On a side note, is it possible to copy a description to destination account during the import process (from Salt Edge) without adding rules for each one to do the job?

**Reply by JC5** [reply]
Nope
